clicmd

```diff
@@ -74,6 +74,18 @@ if [[ ! -f "${REAL_CMD}" ]]; then
   exit 2
 fi
 
+# @brief log audit message
+# @param message
+function log_audit()
+{
+if [ -n "${4-}" ]; then
```

Comment:
It seems to me that the check for the presence of `${1}` and `${2}` is wanted to prevent illegal usage.

Reply:
The `-` mitigates that and produces an empty string if `${1}`, `${2}` or `${3}` aren't present.

Reply:
Yes, and the CLI will keep silence if somebody calls this function with the wrong arguments.

Reply:
@nest1ing, there is just one caller, and the parameters are always there.